extensions/datastores/accumulo/src/main/java/mil/nga/giat/geowave/datastore/accumulo/operations/config/AccumuloRequiredOptions.java

```diff
@@ -25,25 +27,27 @@
 	@Parameter(names = {
 		"-z",
 		"--" + ZOOKEEPER_CONFIG_KEY
-	}, description = "A comma-separated list of zookeeper servers that an Accumulo instance is using", required = true)
+	}, description = "A comma-separated list of zookeeper servers that an Accumulo instance is using", converter = RequiredFieldConverter.class)
```

Comment:
seems like one of the few places we use this.  I understand it gives different functionality (prompting a user).  Do we want this functionality to be consistently used throughout?  Just seems like we should either do this for other current required options or not, or have some logic for when to use this vs. the jcommander annotation.